### Same "cl:variables:star-load-pathname-star.diff"
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:variables:star-load-truename-star.txt	2017-04-27 21:40:24.000000000 +0100
+++ saved-pages-clus/cl:variables:star-load-truename-star.txt	2017-04-27 21:40:24.000000000 +0100
@@ -1,17 +1,38 @@
 Variable *LOAD-PATHNAME*, *LOAD-TRUENAME*  
 Value Type
  
 The value of *load-pathname* must always be a pathname or nil. The value of *load-truename* must always be a physical pathname or nil. 
 Initial Value
  
 nil. 
 Description
  
 During a call to load, *load-pathname* is bound to the pathname denoted by the the first argument to load, merged against the defaults; that is, it is bound to (pathname (merge-pathnames filespec)). During the same time interval, *load-truename* is bound to the truename of the file being loaded. 
 At other times, the value of these variables is nil. 
 If a break loop is entered while load is ongoing, it is implementation-dependent whether these variables retain the values they had just prior to entering the break loop or whether they are bound to nil. 
 The consequences are unspecified if an attempt is made to assign or bind either of these variables. 
-Examples None.
+Examples
+None.
  
 Affected By
  
@@ -15,8 +36,21 @@
  
 Affected By
  
 The file system. 
 See Also
  
-load 
-Notes None.
\ No newline at end of file
+ Function LOAD
+Notes
+None.
+\
\ No newline at end of file
